vsphere_server_plugin/server.py

```diff
@@ -289,13 +344,26 @@ def get_state(server_client, **kwargs):
         else:
             ctx.logger.debug('Public IP check not required for {server}'
                              .format(server=server))
+            # Ensure the property still exists
+            ctx.instance.runtime_properties[PUBLIC_IP] = None
+
+        message = 'Server {name} has started'
+        if manager_network_ip:
+            message += ' with management IP {mgmt}'
+        if len(public_ips) > 0:
+            public_ip = public_ips[0]
+            if manager_network_ip:
+                message += ' and'
+            message += ' public IP {public}'
+        else:
+            public_ip = None
+        message += '.'
```

Comment:
Yeah, I just feel like I'd be recreating the logic if I tried to convert it into a single template because it'd end up:
"{stuff}{possible_conjuction}{more stuff}{possible_conjuction}"

And then all the same/similar logic would be needed to determine whether to make possible_conjuction " and " or ""